journal/updated/01-journal.adoc

```diff
@@ -153,82 +159,84 @@ or people of lower social or economic position.
+which you have heard of me.`" I asked him whether that was scripture;
+he asked me whether I would deny it; I told him he must to prove it.
+Then he took the bible out of my hand, and turned to 2 Tim. 1:13, which he read,
+and told me that "`Hold fast`" there was a command, which I knew very well,
+both the scripture and the command. But to prove him further,
+I desired him to read a little more of that chapter, both before and after,
+which he freely did, and asked me why I asked him to do so.
+I told him that we heard the Quakers denied the scripture,
 and that they would not read them.
 He said there were many false reports of them.
-And truly when he read the scripture so readily, I concluded in myself,
+And truly when he read the scripture so readily, I concluded in myself
 that what was reported of them was not true;
 and he saw that he had reached to the witness of God in me.
 Then he exhorted me to take heed to that light that shined in my heart,
-and did show me my vain thoughts,
-and reprove me in secret for every idle word and action; saying
+which showed me my vain thoughts,
+and reproved me in secret for every idle word and action, saying
 that "`that was the true light,
-which enlightens every man that comes into the world;`" and in that light,
-I should see more light, and that would open the scriptures to me,
-and that I should receive a measure of the same spirit that gave them forth.
-Further he told me, It was "`the more sure word of prophecy,
+which enlightens every man that comes into the world.`" And in that light,
+I would see more light, which would open the scriptures to me,
+so that I would receive a measure of the same Spirit that gave them forth.
+He further told me it was "`the more sure word of prophecy,
 unto which I did well if I took heed, as unto a light that shines in a dark place,
-until the day dawn,
+until the day dawn
 and the day-star arise in your hearts,`" 2 Pet.
 1:19. And he spoke much of the inward work,
-and the operation of God`'s Holy Spirit upon the soul;
-recommending me to the "`Grace of God, that brings salvation,--teaching us, that,
+and the operation of God`'s Holy Spirit upon the soul,
+recommending me to the "`Grace of God, that brings salvation, teaching us, that,
 denying ungodliness and worldly lusts, we should live soberly,
-righteously and godly in this present world,`" Tit. 2:11-12. And so he departed from our house,
-and I set him a little along on his way.
-
-Now when I came back from him, the consideration of his words took fast hold on me,
-that I could not go from under them;
-and the more I waited in that light that he recommended me to, the more my former peace,
-and that in which I formerly took comfort, was broken; and herein I came to see,
+righteously and godly in this present world,`" Tit. 2:11-12.
+And so he departed from our house, and I accompanied him for a little while on his way.
+
+Now when I returned from him, the consideration of his words took fast hold on me,
```

Comment:
Maybe: Now when I returned **from walking with** him, the consideration of his words took **deep hold upon** me,